ansible/readme.md

```diff
@@ -84,7 +84,7 @@ ansible-playbook \
   --user root \
   --extra-vars 'hostname=rustls-bench.example.com' \
   --extra-vars 'github_repo_owner=example' \
-  --extra_vars 'github_repo_name=rustls' \
+  --extra-vars 'github_repo_name=rustls' \
```

Comment:
Oops! Good catch, thank you.

FWIW: your commit message has a typo "unerscore" -> "underscore"